ambry-account/src/main/java/com/github/ambry/account/AbstractAccountService.java

```diff
@@ -269,6 +269,17 @@ public boolean removeAccountUpdateConsumer(Consumer<Collection<Account>> account
     return accountUpdateConsumers.remove(accountUpdateConsumer);
   }
 
+  @Override
+  public void addDataset(short accountId, short containerId, Dataset dataset) throws AccountServiceException {
+    throw new UnsupportedOperationException("This method is not supported");
+  }
+
+  @Override
+  public Dataset getDataset(short accountId, short containerId, String accountName, String containerName,
```

Comment:
why do we need account id and account name at the same time?

Reply:
We need the accountId and containerId to get the dataset and I don't want to add account service to mySqlAccountStore, so previously I put the logic into handler to get accountId and containerId. Now I put it into account service logic which simplify the AccountService interface.